changelog/unreleased/fix-limit-search-only-when-scope-is-passed.md

```diff
@@ -0,0 +1,6 @@
+Bugfix: Limit search only when scope is passed
+
+Previously, the search service would limit the search to the according space when searching `/dav/spaces/`.
+This was not correct, as the search should be limited to the according space when a `scope` is passed in the search pattern instead.
```

Comment:
Are we already doing this? Or are we taking away the ability to search in spaces for now?

Reply:
Yes, scope is already working https://github.com/owncloud/ocis/blob/54ed4837ebe83190c68ade42a24068e32524a4d4/services/search/pkg/search/service.go#L105

Also tested locally to verify this and works as expected.

Reply:
@DeepDiver1975 can you please let us know if we would break clients with this change? The topic is: searching for resources using `/dav/spaces/*` endpoint to limit results to a specific space instead of using `scope` in the search pattern.

@kobergj even if clients wouldn't be affected, should we change this then from enhancement to change to mark a breaking change? I kind of dislike bumping the major version to `8` just because of this... alternative is of course to adjust the behaviour to checking if there is actually the space ID passed in the endpoint and only limit it in such cases instead of completely dropping it.

Reply:
Clarified in chat, no breaking change needed and clients are not using this. We should be good to go.